docs/exceptions.rst

```diff
@@ -37,3 +37,8 @@ Exceptions
 
     This is raised when the verify method of a key derivation function's
     computed key does not match the expected key.
+
+
+.. class:: CheckIssuedFail
```

Comment:
Just as a user of `cryptography` I get the impression that "one-off" exceptions are not defined in `cryptography.exceptions` but near their use site, so probably in `cryptography.x509.base` in this case.

Reply:
Agree. Fixed.